release/mixer.sh

```diff
@@ -53,18 +53,34 @@ build_bundles() {
 
     log_line "Building Bundles:"
     log_line
-    sudo_mixer_cmd build bundles
+    local mixer_opts_bundles=""
+    if function_exists sign_update; then
+        mixer_opts_bundles+=" --no-signing"
+    fi
+    # shellcheck disable=SC2086
+    sudo_mixer_cmd build bundles ${mixer_opts_bundles}
     log_line
 }
 
 build_update() {
     local mix_ver="$1"
 
     section "Build 'Update' Content"
+    local mixer_opts_update=""
     if "${MIN_VERSION}"; then
-        sudo_mixer_cmd build update --skip-format-check --min-version="${mix_ver}"
-    else
-        sudo_mixer_cmd build update --skip-format-check
+        mixer_opts_update+=" --min-version=${mix_ver}"
+    fi
+    if function_exists sign_update; then
```

Comment:
blank line before if

Reply:
I added one after the section so all of the option building code is grouped together?